pages/architecture.md

```diff
@@ -33,6 +33,61 @@ Here's a diagram describing this flow:
 To see if an instrumentation library already exists for your platform, see the
 list of [existing instrumentations]({{ site.github.url}}/pages/existing_instrumentations).
 
+Example flow
```

Comment:
@openzipkin/core might want to take a look at this, too. The attempt here is to show the relationship of sorts of activity in scope of trace instrumentation. We've had plenty of requests to clarify this and probably does help to do so in the architecture pane vs requiring people to look at the instrumenting section.

IOTW, people simply using a zipkin system probably could do well to have a brief overview of the process.

thoughts?